core/google/cloud/future/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017, Google Inc.
```

Comment:
No, the interfaces an behavior are significantly different. Unless @lukesneeringer feels strongly, I will not make a backwards-compatible shim.